### ISSUE TITLE
[Bug 🐞]: Warnings from Module Manager for kerbalism.cfg

### ISSUE THREAD

**Issue by AmanitaVerna** [zer0Kerbal/MiniSampleReturnCapsule#71]
### Brief description of your issue

I just installed Mini Sample Return Capsule and have Kerbalism installed, and while loading KSP, Module Manager reported that there were 4 warnings. I checked its MMPatch.log and found the warnings were coming from Mini Sample Return Capsule's kerbalism.cfg:
```
[WRN 11:20:15.087] more than one pass specifier detected, ignoring all but the first: MiniSampleReturnCapsule/Compatibility/Kerbalism/@KERBALISM_HDD_SIZES:BEFORE[KerbalismDefault]:NEEDS[MiniSampleReturnCapsule,FeatureScience]:FOR[MiniSampleReturnCapsule]
[WRN 11:20:15.088] more than one pass specifier detected, ignoring all but the first: MiniSampleReturnCapsule/Compatibility/Kerbalism/@part[msrc-cargobay]:NEEDS[MiniSampleReturnCapsule,FeatureScience]:AFTER[KerbalismDefault]:FOR[MiniSampleReturnCapsule]
[WRN 11:20:15.088] more than one pass specifier detected, ignoring all but the first: MiniSampleReturnCapsule/Compatibility/Kerbalism/@part[msrc-parachute]:NEEDS[MiniSampleReturnCapsule,FeatureScience]:AFTER[KerbalismDefault]:FOR[MiniSampleReturnCapsule]
[WRN 11:20:15.088] more than one pass specifier detected, ignoring all but the first: MiniSampleReturnCapsule/Compatibility/Kerbalism/@part[msrc-parachute]:NEEDS[MiniSampleReturnCapsule]:FOR[KerbalismDefault]:FOR[MiniSampleReturnCapsule]
```

In the first three warnings, the FOR specifier looks superfluous.
In the fourth warning, there's two FOR specifiers and one refers to KerbalismDefault. I assume the fix for this line is removing the FOR[KerbalismDefault], since you're only supposed to use FOR specifiers for the mod the cfg file is in.

### Steps to reproduce

1. Install Mini Sample Return Capsule.
2. Install Kerbalism? I don't know if it's necessary, since MM is complaining about syntax errors.
3. Start KSP, see if there's a message about warnings. If so, search your KSP folder\Logs\MMPatch.log for '[WRN'.

### Expected behavior

No warnings from Module Manager

### Actual behavior

Warnings from Module Manager

### Environment

```shell
Mod: 1.4.2, according to the KSP Add-on Version Checker. CKAN thinks it is 1.4.1.0 for some reason, but according to the changelogs, there was no kerbalism.cfg in 1.4.1.0.
KSP: 1.12.3.3173
Download/Installation: CKAN
KSP.log: The thing to drop a KSP.log in appears to be missing from this form. (Unless I'm failing to see it because of how little sleep I got last night.)
```

### How did you download and install this?

CKAN (auto install with dependencies)

**Comment by zer0Kerbal** [comment]
thank you @AmanitaVerna !

new release pending.

**Comment by zer0Kerbal** [comment]
@AmanitaVerna 

if you want - here should be the fixed file (GameData/MiniSampleReturnCapsule/Compatibility/Kerbalism.cfg)
[Kerbalism.zip](https://github.com/zer0Kerbal/MiniSampleReturnCapsule/files/8387436/Kerbalism.zip)


### Fixed

* [Kerbalism.cfg] v1.0.1.0
  * Thank you [AmanitaVerna]
  * 'more than one pass specifier detected'
  * removed :FOR[MiniSampleReturnCapsule] and kept :BEFORE/AFTER[KerbalismDefault]
  * concerned that calling KerbalismDefault will tell MM that it is installed.
  * fixes #71 - [Bug 🐞]: Warnings from Module Manager for kerbalism.cfg
